app/javascript/packs/application.js

```diff
@@ -17,23 +17,39 @@ import 'slick-carousel'
 //   });
 // });
 
-import 'mapbox-gl/dist/mapbox-gl.css'; // <-- you need to uncomment the stylesheet_pack_tag in the layout!
```

Comment:
please add back

import 'mapbox-gl/dist/mapbox-gl.css'; // <-- you need to uncomment the stylesheet_pack_tag in the layout!
import { initMapbox } from '../plugins/init_mapbox';
initMapbox();